pages/ucsd/index.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import fetch from 'isomorphic-unfetch';
+
+import { withStyles } from '@material-ui/core/styles';
+
+import GraphViewAssembly from '../../components/graph/GraphViewAssembly';
+import Header from '../../components/Header';
+
+/**
+ * Define the style of components on this page
+ * @param theme
+ * @return {object}
+ */
+const styles = theme => ({
+  wrapper: {
+    'text-align': 'center',
+  },
+  appBar: {
+    position: 'absolute',
+    transition: theme.transitions.create(['margin', 'width'], {
+      easing: theme.transitions.easing.sharp,
+      duration: theme.transitions.duration.leavingScreen,
+    }),
+  },
+});
+
+class GraphPage extends React.Component {
```

Comment:
should be moved into a component

Reply:
Technically the page is a component, but it would be more intuitive to put under pages.